src/components/overlayCookie/overlayCookie.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import 'i18n/i18n';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import styles from './overlayCookie.scss';
+
+class OverlayCookie extends Component {
```

Comment:
Add a story for the component OverlayCookies

Reply:
Done